examples/ui/text_cursor.rs

```diff
@@ -0,0 +1,109 @@
+//! This example illustrates how to create and control a UI text cursor
+
+use bevy::{
+    color::palettes::css::GOLDENROD,
+    diagnostic::FrameTimeDiagnosticsPlugin,
+    prelude::*,
+    ui::widget::{TextCursor, TextCursorStyle, TextCursorWidth},
+};
+
+const CURSOR_WIDTH: f32 = 4.;
+
+fn main() {
+    App::new()
+        .add_plugins((DefaultPlugins, FrameTimeDiagnosticsPlugin::default()))
+        .add_systems(Startup, setup)
+        .add_systems(Update, move_cursor)
+        .run();
+}
+fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
+    // UI camera
+    commands.spawn(Camera2d);
+    // Text with one section
```

Comment:
Some of these comments, presumably from another example are a little odd in this context.